compiler/sem/passes.nim

```diff
@@ -84,13 +85,26 @@ proc closePasses(graph: ModuleGraph; a: var TPassContextArray) =
       m = graph.passes[i].close(graph, a[i], m)
     a[i] = nil                # free the memory here
 
-proc processTopLevelStmt(graph: ModuleGraph, n: PNode, a: var TPassContextArray): bool =
-  # this implements the code transformation pipeline
-  var m = n
-  for i in 0..<graph.passes.len:
+proc processTopLevelStmt(
```

Comment:
Nice, this no longer hurts me when I read it. Thank you for putting an end to the violence.